### DISCUSSION TITLE
whats the alternative for forceAppLaunch ?

### DISCUSSION THREAD

**Discussion by HamzaMiz** [appium/appium#17839] [Q&A]
Hello ,

after upgrading to appium 2.0 , the app wont launch with cap anymore , and i dont know to do it with caps , is there any cap to replace it or i need to use applaunch method to do it ?

Thanks

**Comment by jlipps** [comment]
there aren't enough details here to answer the question. what were you doing before exactly? and what doesn't work now?

**Comment by HamzaMiz** [comment]
Hello , thanks for reply , i m talking about the capability forceAppLaunch which is deprecated and not recognized by appium 
what's the new way to force launch the app before the test ?

**Reply by jlipps** [reply]
Starting the test session will already launch the app. What are you trying to do where this doesn't work?

**Reply by HamzaMiz** [reply]
morning , thats my problem actually , since i started using appium 2.0 , when i launch the test session , the app will not be launched statys at background , and in the logs , hes already trying to find the elements

**Reply by jlipps** [reply]
then submit an issue with the full appium logs. appium is supposed to launch the app. you might also ensure that the app isn't already running by using a different reset strategy.
